app/controlplane/pkg/auditor/events/casbackend.go

```diff
@@ -47,7 +47,7 @@ type CASBackendBase struct {
 	CASBackendName string     `json:"cas_backend_name,omitempty"`
 	Provider       string     `json:"provider,omitempty"`
 	Location       string     `json:"location,omitempty"`
-	Default        bool       `json:"default,omitempty"`
+	Default        bool       `json:"default"`
```

Comment:
I've removed all `omitempty` from the bool fields so they are properly serialized in JSON when stored in the database with the proper value.